src/platform/client/client_platform_linux.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2019 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include <multipass/cli/client_platform.h>
+
+#include <QProcess>
+
+namespace mp = multipass;
+namespace mcp = multipass::cli::platform;
+
+void mcp::open_multipass_shell(const QString& instance_name)
+{
+    QProcess::startDetached("xterm", {"-e", "multipass", "shell", instance_name});
```

Comment:
Add a `-T <instance_name>` so the title is set to the instance name by default? Also `-j` might be desirable.

Reply:
Ack

Reply:
Ok, I tried `-T <instance_name>` and the instance name is in the title of the `xterm` window until bash is up and running, and then the title switches to `multipass@<instance_name>: ~` as before.

Reply:
Yeah and that is fine, but if we don't do that, the whole cmdline will be shown (including `|| read` when you add it) in case the command fails.